graphql/version_resolver.go

```diff
@@ -160,6 +161,31 @@ func (r *versionResolver) ExternalLinksForMetadata(ctx context.Context, obj *res
 	return externalLinks, nil
 }
 
+// GeneratedTaskCounts is the resolver for the generatedTaskCounts field.
+func (r *versionResolver) GeneratedTaskCounts(ctx context.Context, obj *restModel.APIVersion) (map[string]interface{}, error) {
+	versionID := utility.FromStringPtr(obj.Id)
+	v, err := model.VersionFindOneId(*obj.Id)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while finding version with id: `%s`: %s", versionID, err.Error()))
+	}
+	if v == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Version does not exist: `%s`", versionID))
+	}
```

Comment:
nit. formatting and quotes
```suggestion
	if err != nil {
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding version with id '%s': %s", versionID, err.Error()))
	}
	if v == nil {
		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("version '%s' does not exist", versionID))
	}
```